apps/scratch/src/hooks/useIframeParent.ts

```diff
@@ -0,0 +1,172 @@
+import { useCallback, useEffect, useRef, useState } from "react";
+import {
+  AppIFrameMessage,
+  AppIFrameRequest,
+  AppIFrameResponse,
+} from "../../../../frontend/src/types/app-iframe-message";
```

Comment:
oops, that was automatically done by the IDE but you are right ^^

Reply:
The problem is that we need a alias because this comes from a different project ^^ So maybe just change this once we have our external library for the iframe communication?